charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -10,10 +10,10 @@
 [WARNING] "secretMounts" parameter is deprecated, please use "extraVolumes" or "extraVolumesMounts" instead.
 {{ end }}
 
-{{- if not (eq (toString .Values.mode) "<nil>") }}
+{{- if not .Values.mode }}
```

Comment:
Makes sense. I thought we didn't define default mode in values.yaml at all

Reply:
values schema didn't like the null as a possible string enum, so its default is "", which is also false.